src/containers/Bounty/Bounty.module.scss

```diff
@@ -66,14 +67,12 @@
 
 .stage {
   position: absolute;
-  right: 0;
-  top: 0;
+  left: 30%;
```

Comment:
We should move the stage pill into the `BountyPayout` component -- so you'll also need a new prop to accept the bounty stage.

Reply:
We'd need to rethink `BountyPayout` component then. If we move the pill to it, it'd have more responsibility than just bounty payouts.

Reply:
I agree here, this is one component we'll likely need to refactor a bit, since it encompasses the status now instead of just the value.